### PR TITLE
Login migration upgrade guides and error handling

### PR DESCRIPTION
**Pull request by Simon-Laux**
- Add information about changed behavior. Also it shouldn't crash anymore if the migration failed, instead it opens an error dialog
- show error dialogs on uncaught errors, in hope to get more less technical people to report crashes

### CODE REVIEW

File: src/main/logins.ts [hunk 1]
```diff
@@ -33,8 +34,23 @@ async function migrate(dir: string) {
...
+        await fs.move(join(dir, basename(account.path)), join(dir, newFolder))
+      } catch (error) {
+        log.error(
+          "Moving failed, make sure you don't have multiple account folders for the same account!",
+          error
+        )
+        dialog.showErrorBox(
+          'Account migration',
+          `Migration of ${account.addr} failed,\n` +
+            "make sure you don't have multiple account folders for the same account!\n" +
```
Comment (Jikstra) [inline src/main/logins.ts:47]: make sure you don't have the same e-mail adress multiple times

Comment (Simon-Laux) [reply src/main/logins.ts:47]: I see your point but your proposed solution misses the point with the account folders

Comment (Jikstra) [reply src/main/logins.ts:47]: `Please make sure that you don't have multiple account folders for the same e-mail address`

Comment (Simon-Laux) [inline src/main/logins.ts:47]:
```suggestion
            "Please make sure that you don't have multiple account folders for the same e-mail address!\n" +
```

File: src/main/logins.ts [hunk 2]
```diff
@@ -33,8 +34,23 @@ async function migrate(dir: string) {
         'accounts',
         escapeEmailForAccountFolder(account.addr)
       )
-      await fs.move(join(dir, basename(account.path)), join(dir, newFolder))
-      // Backwards compatibility
+      try {
+        await fs.move(join(dir, basename(account.path)), join(dir, newFolder))
+      } catch (error) {
+        log.error(
+          "Moving failed, make sure you don't have multiple account folders for the same account!",
```
Comment (Simon-Laux) [inline src/main/logins.ts:41]:
```suggestion
          "Moving failed, make sure you don't have multiple account folders for the same e-mail address!",
```

**Pull request merged**